src/components/admin/BlockSuggestionComponent.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { NavLink } from 'react-router-dom'
+
+class BlockSuggestionComponent extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+    }
+    this.block = this.block.bind(this)
+  }
+
+  block(e) {
+    this.setState({
+      blockSuccess: true
+    })
+  }
+    render() {
+
+        let notFound = this.props.data.influencername === 'No matching influencers'
+        let profileImage
+        if (!notFound) {
+            profileImage = <img src={'https://pbs.twimg.com/media/DoGZoooXgAACCV3.jpg'} />
+        } else {
+            profileImage = ''
+        }
```

Comment:
Do you really need that else? Why not set `profielImage = ''` from the beginning?